content/reference/apis/configuration_variables/secrets.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Secrets'
+weight: 2
+---
+
+Once a secret is set using [wrangler](/tooling/wrangler/secret) or in the UI, it becomes available to your Worker script as its string value. 
+
+Example where `SECRET` is set  and then set as a header `Auth` in a fetch.
```

Comment:
it's probably worth putting in an example line from `wrangler.toml` here to show what it would be and what the repsonse would look like